pmm-app/src/pmm-dbaas/components/DBCluster/AddDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -34,7 +34,13 @@ export const DBClusterAdvancedOptions: FC<FormRenderProps> = ({
   const { required, min } = validators;
   const { change } = form;
   const nodesValidators = [required, min(MIN_NODES)];
-  const resourcesValidators = [required, min(MIN_RESOURCES)];
+  const diskValidators = [required, min(MIN_RESOURCES)];
+  const resourcesValidators = [
+    required,
+    min(0.1),
```

Comment:
I saw in other PR a constant for this. Can it be used here too?

Reply:
Yes, thanks it's in PR for PMM-7018

Reply:
removed, was added by mistake